### PR TITLE
galleryimg command

### PR DESCRIPTION
**Pull request by apozsuse**
Command for galleryimg management

- Includes 3 subcommands for existence check, creation and deletion of a galleryimg
- Update of the README.md and the unit tests accordingly

# Remarks

Due to the Azure SDK replying always None for the result of the async delete operation, it's not possible to provide more accurate info for the user about the image deletion (without implementing some logic). [This issue](https://github.com/SUSE-Enceladus/azure-img-utils/issues/37) has been created to revisit this behavior in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +276,99 @@ For more information about the image delete command see the help message:
...
 
+## galleryimg command
+
+Group of commands for gallery image management.
+
+### exists subcommand
+
+This subcommand allows the user to check if an image version exists in a
+ gallery.
+The subcommand is *azure-img-utils galleryimg exists*.
```
Comment (smarlowucf) [inline README.md:287]: Although the command will be quite long I think we need to be clear on what the sub-command is for. Gallery Images are a different topic. Here we are only handling Gallery Image Versions. Thus I'd suggest the sub-command like gallery-img-versions or gallery-image-versions.

**Review by smarlowucf — CHANGES_REQUESTED** [review]
Looks good, although I think we want to be clear with naming.

File: azure_img_utils/cli/galleryimg.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    help='Name of the gallery image to check.'
+)
+@click.option(
+    '--gallery-name',
+    type=click.STRING,
+    required=True,
+    help='Name of the gallery to check image existence.'
+)
+@click.option(
+    '--image-version',
```
[the remark this answers is not in the document]
Comment (apozsuse) [reply azure_img_utils/cli/galleryimg.py]: Done!

File: README.md [hunk 3]
```diff
@@ -276,6 +275,106 @@ For more information about the image delete command see the help message:
...
+ help message:
+
+```shell
+$ azure-img-utils gallery-image-version exists --help
+```
+
+### create subcommand
+
+This subcommand allows the user to create a gallery image version in a gallery based
+ in one blob.
```
Comment (rjschwei) [inline README.md:316]: s/in one/on a/

File: azure_img_utils/cli/gallery_image_version.py [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+@click.pass_context
+def delete(
+    context,
+    gallery_name,
+    gallery_image_name,
+    gallery_image_version,
+    **kwargs
+):
+    """
+    Deletes an gallery image if the image exists in the gallery
```
Comment (rjschwei) [inline azure_img_utils/cli/gallery_image_version.py:238]: s/an/a/

**Review by rjschwei — CHANGES_REQUESTED** [review]

**Review by rjschwei — APPROVED** [review]

**Review by smarlowucf — APPROVED** [review]

**Pull request merged**